### PR TITLE
Feature/df 2200 dublicate jobs

### PR DESCRIPTION
**Pull request by c-linse**
In /alv-portal/alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.component.spec.ts unit tests takes place already for the form itself. A comprehensive unit testing for the factory can be made though. This is out of scope of the ticket and should be done in a different story. What do you guys think?

### CODE REVIEW

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form-value-factory.ts [hunk 1]
```diff
@@ -32,6 +34,10 @@ export class JobPublicationFormValueFactory {
       const duplicatedJobPublicationFormValue = jobPublicationFormMapper.mapToJobPublicationFormValue(initialFormValueConfig.jobAdvertisement, currentLanguageIsoCode);
       if (initialFormValueConfig.companyContactTemplateModel) {
         duplicatedJobPublicationFormValue.contact = mapToContactFormValue(initialFormValueConfig.companyContactTemplateModel, currentLanguageIsoCode);
+        duplicatedJobPublicationFormValue.company = mapToCompanyFormValue(initialFormValueConfig.companyContactTemplateModel);
+        duplicatedJobPublicationFormValue.company = mapToCompanyFormValue(initialFormValueConfig.companyContactTemplateModel);
+        duplicatedJobPublicationFormValue.surrogate=false;
```
Comment (birom) [inline alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form-value-factory.ts:39]: Does not tslint complain about this line?
Should be ``duplicatedJobPublicationFormValue.surrogate = false``

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form-value-factory.ts [hunk 2]
```diff
@@ -32,6 +34,10 @@ export class JobPublicationFormValueFactory {
       const duplicatedJobPublicationFormValue = jobPublicationFormMapper.mapToJobPublicationFormValue(initialFormValueConfig.jobAdvertisement, currentLanguageIsoCode);
       if (initialFormValueConfig.companyContactTemplateModel) {
         duplicatedJobPublicationFormValue.contact = mapToContactFormValue(initialFormValueConfig.companyContactTemplateModel, currentLanguageIsoCode);
+        duplicatedJobPublicationFormValue.company = mapToCompanyFormValue(initialFormValueConfig.companyContactTemplateModel);
+        duplicatedJobPublicationFormValue.company = mapToCompanyFormValue(initialFormValueConfig.companyContactTemplateModel);
```
Comment (birom) [inline alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form-value-factory.ts:38]: Why is ``company`` twice there?

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form-value-factory.ts (hunk 1 above)
Comment (c-linse) [reply alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form-value-factory.ts]: Linter made problems, this suggestion works!

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form-value-factory.ts (hunk 2 above)
Comment (c-linse) [reply alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form-value-factory.ts]: Sorry, my bad. I've removed the duplicated line.

**Pull request merged**